app/src/components/Footer.tsx

```diff
@@ -16,16 +16,16 @@ const Footer = () => (
       <Link to="" tooltip="Twitter">
         <FaTwitter />
       </Link>
-    </div>
+    </Flex>
 
     <div>
       A project of the{" "}
-      <Link to="https://www.thekrishnanlab.org" noIcon={true}>
+      <Link to="https://www.thekrishnanlab.org" noIcon>
```

Comment:
Sorry, didn't catch this until now; you might want to update this to jravilab.org.